pkg/ottl/ottlfuncs/README.md

```diff
@@ -1068,6 +1069,32 @@ Examples:
 
 - `Split("A|B|C", "|")`
 
+### String
+
+`String(value)`
+
+The `String` Converter converts the `value` to string type.
+
+The returned type is `string`.
+
+- float64. The function returns the `value` as a string type.
+- string. The function returns the `value`, even if it's empty.
+- bool. The function returns true/false as "true"/"false".
+- int64. The function returns the `value` as a string type.
+- struct. The function returns an empty struct as "{}"
```

Comment:
This isn't true.  Checkout https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/16378865457151125248414e887edf62a507abae/pkg/ottl/expression.go#L389-L422 for how the conversion happens

Reply:
i updated that in latest commit